tools/lsp/build.rs

```diff
@@ -2,5 +2,13 @@
 // SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
 
 fn main() {
+    // Make the compiler handle ComponentContainer:
+    // TODO: Remove if ComponentContainer anc component-factory no longer need this!
+    std::env::set_var("SLINT_ENABLE_EXPERIMENTAL_FEATURES", "1");
+    slint_build::compile_with_config(
+        "ui/main.slint",
+        slint_build::CompilerConfiguration::new().with_style("fluent-light".into()),
+    )
+    .unwrap();
     println!("cargo:rustc-env=SLINT_ENABLE_EXPERIMENTAL_FEATURES=1");
```

Comment:
I guess this is no longer needed then.

Reply:
Why do the rust compiler need to see it?
If we're not using the slint! macro anymore, it shouldn't be read by anything else.

Reply:
But this env variable is only used by the slint compiler, through slint-build.
Rustc only need this if using the slint! Macro which you are not using anymore.

> my code interacts with that,

But that doesn't need this env variable. Or where does it need it?